docs/user/openstack/README.md

```diff
@@ -104,6 +104,8 @@ The bootstrap node is a temporary node that is responsible for standing up the c
 
 If Swift is available in the cloud where the installation is being performed, it is used as the default backend for the OpenShift image registry. At the time of installation only an empty container is created without loading any data. Later on, for the system to work properly, you need to have enough free space to store the container images.
 
+**Note:** If you are deploying a cluster in an Availability Zone where Swift isn't available, but where Cinder is, it is highly suggested to deploy the Image Registry in Cinder, which is aware of the topology and will schedule the volumes into the right zone.
```

Comment:
Can we move the note below, after we've stated that Cinder is the fallback when Swift is not available? I think where it's placed now it breaks the flow. For example the line below says "In this case" which could now refer to when swift is available or not, depending if you read the note.

I would also perhaps add a blurb about how to explicitly chose cinder (remove the `swiftoperator` permission).

Also, have we confirmed that the image-registry volume is always created in the same zone as the nodes, when only one zone is specified? If not we should probably add instruction to move the image-registry on day2. Doesn't have to be in this PR.

Reply:
ack, will do

Reply:
I modified the content a bit but I want a review from @Fedosin to confirm that the Image Registry will always request a volume in the right AZ, with proper technical explanation.

Reply:
I got it from @gnufied:
```
a pod with pvc is scheduled in nova zone - us-east-1c, then volume is also created with same AZ. it is expected that cinder and nova zones match at that point
```